quiz.rb

```diff
@@ -1,20 +1,60 @@
+require 'pry'
 # Question 1
 # Create a class called Dog. It should have, as attributes, name, breed, and age.
 # Make all of those attributes read-only.
 
+class Dog
+  def initialize(name, breed, age)
+    @name = name
+    @breed = breed
+    @age = age
+  end
+  attr_reader :name, :breed, :age
```

Comment:
Small point - it's convention to put `attr_reader` before the `initialize` method.